dev/devicelab/lib/framework/devices.dart

```diff
@@ -568,18 +564,18 @@ class AndroidDevice extends Device {
   @override
   Future<Map<String, dynamic>> getMemoryStats(String packageName) async {
     final String meminfo = await shellEval('dumpsys', <String>['meminfo', packageName]);
-    final Match match = RegExp(r'TOTAL\s+(\d+)').firstMatch(meminfo);
+    final Match? match = RegExp(r'TOTAL\s+(\d+)').firstMatch(meminfo);
     assert(match != null, 'could not parse dumpsys meminfo output');
     return <String, dynamic>{
-      'total_kb': int.parse(match.group(1)),
+      'total_kb': int.parse(match!.group(1)!),
     };
   }
 
   @override
   bool get canStreamLogs => true;
 
   bool _abortedLogging/*!*/ = false;
```

Comment:
What's up with these commented out `!`s and `?`s?  Notes for migration to null safety or something?  Should this one be removed?

Reply:
Yeah, it looks like some of this was at one time ported in some fashion to null safety but then commented out. I got most of them, but missed this one. I will fix it. Thanks.